aiida_restapi/routers/auth.py

```diff
@@ -105,32 +99,35 @@ async def get_current_user(token: str = Depends(oauth2_scheme)):
 
 
 async def get_current_active_user(
-        current_user: User = Depends(get_current_user)):
+    current_user: User = Depends(get_current_user),
+) -> User:
     if current_user.disabled:
```

Comment:
@ltalirz the final error I'm getting is:

```
aiida_restapi/routers/auth.py:104: error: "User" has no attribute "disabled"  [attr-defined]
```

This `disabled` attribute comes from the aiida ORM object right, so does it not also need to be added as a field on the pydantic model?

Reply:
That should be `UserInDB` instead of `User` I think

Reply:
done